pvacseq/lib/main.py

```diff
@@ -128,6 +128,12 @@ def main(args_input = sys.argv[1:]):
              + "Needs to be an even number.",
     )
     parser.add_argument(
+        "-d", "--downstream-sequence-length",
+        default='1000',
+        help="Cap to limit the downstream sequence length for frameshifts when creating the fasta file. "
+            + "Use 'full' to include the full downstream sequence. Default: 1000"
+    )
```

Comment:
Oh wait, you're using `full` as a special value.  Nevermind!  Though some people like `-1` for that sort of thing.